LottoDairy/CalendarScene/Views/HeaderView/ScopeButton.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  ScopeButton.swift
+//  LottoDairy
+//
+//  Created by Brody on 2023/09/08.
+//
+
+import UIKit
+
+protocol ScopeChangeDelegate: AnyObject {
+    func changeState()
+}
+
+final class ScopeButton: UIView {
+    
+    private let monthLabel: UILabel = {
+        let label = UILabel()
+        label.text = "월간"
+        label.font = .gmarketSans(size: .subheadLine, weight: .bold)
+        label.textColor = .white
+        label.textAlignment = .center
+        label.translatesAutoresizingMaskIntoConstraints = false
+        
+        return label
+    }()
```

Comment:
```suggestion
    private let monthLabel: UILabel = {
        let label = GmarketSansLabel(text: "월간", size: .subheadLine, weight: .bold)
        label.translatesAutoresizingMaskIntoConstraints = false
        return label
    }()
```

monthLabel, weekLabel 모두 설정한 속성이 GmarketSansLabel에 정의된 속성과 동일해 보여요! 해당 UI Component를 사용해보는건 어떨까요?